code/modules/clothing/glasses/_glasses.dm

```diff
@@ -38,8 +42,12 @@
 
 /obj/item/clothing/glasses/weldingvisortoggle(mob/user)
 	. = ..()
+	alternate_worn_layer = up ? ABOVE_BODY_FRONT_HEAD_LAYER : null
 	if(. && user)
 		user.update_sight()
+		if(iscarbon(user))
+			var/mob/living/carbon/carbon_user = user
+			carbon_user.head_update(src, forced = 1)
```

Comment:
```suggestion
			carbon_user.head_update(src, forced = TRUE)
```

Reply:
Good catch, I didn't even think about it, I think I just kinda copied this from somewhere else.